src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/manual/ManualSyncRunner.java

```diff
@@ -81,9 +81,11 @@ public void run(ProgressStatus progressStatus) {
         progressStatus.setCurrent(0);
 
         List<Pair<Element, ObjectNode>> clientElements = new ArrayList<>(rootElements.size());
+        List<String> clientIdsVisited = new ArrayList<>(rootElements.size());
```

Comment:
Let's make this a `Set` (`HashSet`) so https://github.com/Open-MBEE/mdk/pull/222/files#diff-f9653fe22a4facbe1b3b5d3f6e22b5b5e003b0f693b24bd219c823a7fc9f8eafR138 is `O(1)` instead of `O(n)`.